content/en/docs/collector/deployment/single-writer-principle.md

```diff
@@ -0,0 +1,57 @@
+---
+title: The Single Writer Principle
```

Comment:
```suggestion
title: The Single Writer Principle
```

How can we better contextualize what this doc is about when it comes to Collector deployments? Think of a nav title that would fit the progression in the current section. Why should someone know about this principle?

Reply:
Also, could this be a section in an existing doc rather than its own doc?

Reply:
Ah, right. It might make sense in `collectors/scaling` as a subsection. What do you think?